src/main/java/com/microservices/choreographyorchestrator/util/DtoToDataUtil.java

```diff
@@ -0,0 +1,53 @@
+package com.microservices.choreographyorchestrator.util;
+
+import com.microservices.choreographyorchestrator.domain.MetaNode;
+import com.microservices.choreographyorchestrator.domain.meta.*;
+import com.microservices.choreographyorchestrator.dto.MetaInformationDto;
+import com.microservices.choreographyorchestrator.dto.meta.CompensationInfoDto;
+import com.microservices.choreographyorchestrator.dto.meta.FailExecutionInfoDto;
+import com.microservices.choreographyorchestrator.dto.meta.RetryInfoDto;
+import com.microservices.choreographyorchestrator.dto.meta.SuccessExecutionInfoDto;
+
+public class DtoToDataUtil {
```

Comment:
We can use so called `ModelMapper` instead. It will automatically map the entities to dtos and vise versa

Reply:
You can leave this as is for now but create a task for you for refactoring this to `ModelMapper`. And your next PR should include this changes

Reply:
Have you created the task?